javascript/src/URDFLoader.js

```diff
@@ -73,7 +73,11 @@ class URDFLoader {
         const workingPath = THREE.LoaderUtils.extractUrlBase(urdf);
         const urdfPath = this.manager.resolveURL(urdf);
 
-        options = Object.assign({ workingPath }, options);
+        options = Object.assign({
+            workingPath,
+            visualEnabled: true,
```

Comment:
Option defaults (aside from `workingPath` because it's only available when loading from URL) should only be defined down in the `parse` function like the rest of the options.